### ISSUE TITLE
Abnormal Lung Auscultation Finding

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7118]
**Preferred term label:**
abnormal lung auscultation finding

**Synonyms**
abnormal lung auscultation

**Definition (free text, please give PubMed ID)**
Abnormal respiratory sounds identified during lung auscultation, usually performed using a stethoscope. [PMID: 26229557]

**Parent term (use hpo.jax.org/app)** _(updated 2021-09-13)_
~HP:0002088 - Abnormal lung morphology~
HP:0012252 - Abnormal respiratory system morphology 
_And/Or:_ HP:0030829 - Abnormal breath sound

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Comment by pnrobinson** [comment]
Auscultation is listening to the sounds of the body during a physical examination (not necessarily with a stethoscope)
Therefore, this is almost 100% synonymous with Abnormal breath sound. The HPO typically does not include separate terms for phenotypes based on how they are measured, this there would notbe a separate term for rhonci diagnosed by stethoscope.
I will add this as a related synonym to HP:0030829 - Abnormal breath sound
